bin/nlg-eval

```diff
@@ -138,6 +138,7 @@ def setup(ctx, param, value):
         from nlgeval.word2vec.generate_w2v_files import generate
         z = ZipFile(os.path.join(data_path, 'glove.6B.zip'))
```

Comment:
```python
        with ZipFile(os.path.join(data_path, 'glove.6B.zip')) as f:
            z.extract('glove.6B.300d.txt', data_path)
```

Reply:
I think we didn't do this before so that it works with Python 2.6 and below.

Reply:
oh i never wrote anything for python<2.7, I wouldn't know what works there :-) ok then!

Reply:
I remembered looking this up for zip files and finding this: https://docs.python.org/2/library/zipfile.html#zipfile.ZipFile